client/css/style.css

```diff
@@ -815,25 +815,30 @@ button {
 #chat .time,
 #chat .from,
 #chat .text {
-	display: table-cell;
+	display: block;
 	padding: 2px 0;
-	vertical-align: top;
+	flex: 0 0 auto;
 }
 
 #chat .time {
 	color: #ddd;
 	text-align: right;
-	max-width: 46px;
-	min-width: 46px;
+	width: 46px;
 }
 
 #chat .from {
 	border-right: 1px solid #f6f6f6;
 	color: #b1c3ce;
 	padding-right: 10px;
 	text-align: right;
-	max-width: 134px;
-	min-width: 134px;
+	width: 134px;
+	align-self: stretch;
+}
+
+#chat .text {
+	margin: auto;
```

Comment:
Not sure the specific reason for this, but this is adding a top margin when text is one line but "from" goes over multiple lines. Example:

`margin: auto;` | ~~`margin: auto;`~~
--- | ---
<img width="435" alt="screen shot 2016-09-20 at 00 41 59" src="https://cloud.githubusercontent.com/assets/113730/18657762/42430250-7ecb-11e6-8341-c2ab1a0a0acb.png"> | <img width="427" alt="screen shot 2016-09-20 at 00 42 42" src="https://cloud.githubusercontent.com/assets/113730/18657763/42481150-7ecb-11e6-96e9-cba53397b9c9.png">